modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/api/lifecycle/AddEndPointSecurityPerTypeTestCase.java

```diff
@@ -497,6 +496,113 @@ public void testAPIDefinitionImportWithEndpointSecurity() throws Exception {
                 applicationKeyBeanProduction.getConsumerSecret());
     }
 
+    @Test(groups = { "wso2.am"}, description = "Test add OAuth endpoint security for PASSWORD grant type",
+            dependsOnMethods = "testAddEndpointSecurityForOauthForClientCredentialsGrantType")
+    public void testAddEndpointSecurityForOauthForPasswordGrantType() throws Exception {
+
+        String clientCredGrantTypeEndpointSecurityForProductionAndSandboxForPasswordGrantType = "{\n" +
```

Comment:
Better if we can make this string `final`

Reply:
Fixed in https://github.com/wso2/product-apim/pull/13192